packages/core/src/routes/experience/classes/verifications/index.ts

```diff
@@ -138,3 +140,41 @@ export const buildVerificationRecord = (
     }
   }
 };
+
+export const buildUserVerificationRecordById = async (
```

Comment:
nit: Could move these two methods to a new verification library. Decouple the get and get logic from the verification class itself.